practice_independent_challenges/week2/potluck/spec/potluck_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rspec'
+require './lib/dish'
+require './lib/potluck'
+
+describe Potluck do 
+    describe '#initialize' do
+        it 'has a date' do
+            potluck = Potluck.new("7-13-18")
+            expect(potluck.date).to eq "7-13-18"
+        end 
+
+        it 'has no dishes by default' do
+            potluck = Potluck.new("7-13-18")
+            expect(potluck.dishes).to eq []
+        end 
+        
+    end
+
+    describe '#add_dish' do
+        it 'returns a list of dishes' do
+            potluck = Potluck.new("7-13-18")
+            couscous_salad = Dish.new("Couscous Salad", :appetizer)
+            cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+            potluck.add_dish(couscous_salad)
+            potluck.add_dish(cocktail_meatballs)
+            expect(potluck.dishes).to eq ["Couscous Salad", "Cocktail Meatballs"]
+        end
+
+        it 'returns the length of the dishes array' do
```

Comment:
The description in the 'it' statements are not entirely correct. Your describe block is for the #add_dish method, and that method doesn't return the length of dishes or a list of dishes but instead adds dishes to the potluck. Also, you could combine these two into a single test since they are both verifying that the dishes were added to the potluck.

Reply:
Updated the it statement and combined both tests.